client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -207,6 +218,12 @@ const CheckoutThankYou = React.createClass( {
 			);
 		}
 
+		let content = this.productRelatedMessages();
+
+		if ( ! isEmpty( failedPurchases ) ) {
```

Comment:
It'd be nicer if `failedPurchases` was always an array and we checked against it's `.length`.